features/product/viewing_products/viewing_a_products_lowest_price_from_30_days_before_the_discount.feature

```diff
@@ -0,0 +1,76 @@
+        And this product's price was discounted to "$21.00"
+        And this product's discounted price changed to "$30.00"
+        When I check this product's details
+        Then I should see "$21.00" as this product's lowest price from 30 days before the discount
+
+    @todo
+    Scenario: Seeing the lowest price information on a product that has a discount and then it has a more attractive discount change
+        Given the store has a product "Wyborowa Vodka" priced at "$37.00"
+        And this product's price was discounted to "$21.00"
+        And this product's discounted price changed to "$9.00"
+        When I check this product's details
+        Then I should see "$9.00" as this product's lowest price from 30 days before the discount
+
+    @todo
+    Scenario: Seeing the lowest price information on a product that has a discount, over a month passes, and then it has a less attractive discount change
+        Given it is "2023-03-14" now
+        And the store has a product "Wyborowa Vodka" priced at "$42.00"
+        And this product's price was discounted to "$21.00"
+        And it is "2023-04-15" now
+        And this product's discounted price changed to "$33.00"
+        When I check this product's details
+        Then I should see "$33.00" as this product's lowest price from 30 days before the discount
+
+    @todo
+    Scenario: Seeing the lowest price information on a product that has a discount, over a month passes, and then it has a more attractive discount change
+        Given it is "2023-03-14" now
+        And the store has a product "Wyborowa Vodka" priced at "$42.00"
+        And this product's price was discounted to "$21.00"
+        And it is "2023-04-15" now
+        And this product's discounted price changed to "$19.00"
+        When I check this product's details
+        Then I should see "$19.00" as this product's lowest price from 30 days before the discount
+
+    @todo
+    Scenario: Not seeing the lowest price information on a product that had a discount and the discount was removed
+        Given the store has a product "Wyborowa Vodka" priced at "$37.00"
+        And this product's price was discounted to "$21.00"
+        And this product's discount was removed
+        When I check this product's details
+        Then I should not see information about its lowest price
+
+    @todo
+    Scenario: Seeing the lowest price information on a product that had a discount, the discount was removed, the catalog price changed below the discount price, the catalog price changed back to the start value and then the less attractive discount was added
+        And the store has a product "Wyborowa Vodka" priced at "$37.00"
+        And this product's price was discounted to "$21.00"
+        And this product's discount was removed
+        And this product's catalog price was changed to "$10.00"
+        And this product's catalog price was changed back to "$21.00"
```

Comment:
In Sylius `price` === `catalog price`, as there is no such a concept like catalog price. If your intention is to replicate the behaviour of catalog promotions, I would be in favour of setting up simple catalog promotions here for this purpose, just one extra step probably

Reply:
I was referring to the basic price change. For instance, we use such a naming here:
https://github.com/Sylius/PriceHistoryPlugin/blob/main/features/product/managing_product_variants/seeing_correct_catalog_price_history_after_editing_product_variant.feature#L5

Reply:
In the linked case it's used in the feature's description so it's less of a problem, but since we have steps like:

> And the store has a product "Wyborowa Vodka" priced at "$37.00"

> And this product's price was discounted to "$21.00"

with no mention of a `catalog price`, I'd be for leaving it as simply `price`.
Preferably also remove the `catalog` part everywhere else to keep things consistent across the board.